.github/workflows/test-template.yml

```diff
@@ -238,6 +238,70 @@ jobs:
           key: ${{ secrets.BUILDPULSE_ACCESS_KEY_ID }}
           secret: ${{ secrets.BUILDPULSE_SECRET_ACCESS_KEY }}
 
+  #
+  # CLIENT (functional tests with driver adapters)
+  #
+  client-driveradapters:
+    timeout-minutes: ${{ inputs.jobTimeout }}
+    runs-on: ubuntu-latest
+
+    if: ${{ contains(inputs.jobsToRun, '-all-') || contains(inputs.jobsToRun, '-client-') }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        #shard: ['1/6', '2/6', '3/6', '4/6', '5/6', '6/6']
+        #node: [16, 18, 20]
+        provider: ['postgresql'] #'sqlite', 'mysql']
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        env:
+          DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+          DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+        if: "${{ env.DOCKERHUB_USERNAME != '' && env.DOCKERHUB_TOKEN != '' }}"
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - run: docker compose -f docker/docker-compose.yml up --detach postgres neon_wsproxy
+
+      - name: Install & build
+        uses: ./.github/actions/setup
+        with:
+          node-version: ${{ matrix.node }}
+          pnpm-version: ${{ inputs.pnpmVersion }}
+          # Fails if set to true
```

Comment:
Why?

Reply:
Copy-Paste from above :shrug:

Reply:
I tend to think it should be skipped, as the other jobs will type-check already